README.md

```diff
@@ -23,9 +23,9 @@ To make NUR accessible for your login user, add the following to `~/.config/nixp
 ```nix
 {
   packageOverrides = pkgs: {
-    nur = pkgs.callPackage (import (builtins.fetchGit {
-      url = "https://github.com/nix-community/NUR";
-    })) {};
+    nur = import (builtins.fetchTarball "https://github.com/nix-community/NUR/archive/master.tar.gz") {
```

Comment:
Sorry I missed why using callPackage was an issue. Both are fine for me

Reply:
Because when you do `pkgs.callPackage`, this needs to evaluate `pkgs` (to get the attribute), which requires evaluating all overlays and the config (because they're passed to the nixpkgs import`, but those overlays can be defined in all modules. So if you want to `imports = [ nur.repos.infinisil.modules.foo ]`, this gives infinite recursion, because to evaluate the modules it needs to evaluate pkgs!

Reply:
Thanks, that makes a lot of sense